src/components/Main/main.component.js

```diff
@@ -22,15 +22,15 @@ export const MainComponent = ({
         </ImageBackground>
       </Animated.View>
       <Animated.View style={[styles.categories]}>
-        {categories?.data.map((item) => renderCategoryCard(item))}
+        {categories.map((item) => renderCategoryCard(item))}
```

Comment:
why do you render it in scrollview? It's against optimization principles of react native